### PR TITLE
Alcail/config camera button focus

### PR DESCRIPTION
**Pull request by alcail**
# What
allow control bar buttons to be focusable even when disabled for LocalPreview and CallControls in Call composite

# Why
focus was lost on camera button after turning it on due to button being disabled till we get the video stream
bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2562167

# How Tested
ran call sample and navigate through keyboard

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by prprabhu-ms — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/components/CallControls.tsx [hunk 1]
```diff
@@ -145,6 +145,7 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
       showLabel={!compactMode}
       styles={controlButtonBaseStyle}
       {...microphoneButtonStrings}
+      allowDisabledFocus={true}
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:148]: nit: thoughts on having this property as default on the ControlBarButton.tsx:
```
return (
    <ControlButtonTooltip content={tooltipContent} id={tooltipId}>
      <DefaultButton
          allowDisabledFocus={props.allowDisabledFocus ?? true}
```
?
Pros: accessibility by default for customers
Cons: slight divergence from Fluent's default button

Comment (alcail) [reply packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:148]: I was wondering about it too but decided to not go there as it should be a decision on Contoso when using components.
I agree that having it as true by default is a better behaviour, but that's something I can discuss with Alex P to have a better insights

Comment (JamesBurnside) [reply packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:148]: Contoso can still decide, we'd only be setting allowDisabledFocus to be true _by default_ (they can set it to false if they don't want it). I'd still advocate for having it in the control bar button as we claim to be an a11y-centric library in our storybook but then wouldn't be applying a11y specific properties up front (Contoso has to set a quite hidden property to get the a11y feature)

Comment (alcail) [reply packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:148]: true :)

Comment (alcail) [reply packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:148]: I will update the storybook doc to reflect this though

**Review by JamesBurnside — APPROVED** [review]

**Pull request merged**